pkgs/top-level/python-packages.nix

```diff
@@ -8384,13 +8267,15 @@ in {
   };
 
   fastimport = callPackage ../development/python-modules/fastimport { };
- 
+
   feedgen = callPackage ../development/python-modules/feedgen { };
 
   feedgenerator = callPackage ../development/python-modules/feedgenerator {
     inherit (pkgs) glibcLocales;
   };
 
+  s3transfer = callPackage ../development/python-modules/s3transfer { };
```

Comment:
Maybe you didn't mean to move `s3transfer` here from its old place far below?

Reply:
Not intentional. Fixed!